modules/common/src/DatasetInfo.php

```diff
@@ -135,6 +135,34 @@ public function gather(string $uuid) : array {
     return $info;
   }
 
+  /**
+   * Get the current distribution UUID for a dataset.
+   *
```

Comment:
How about adding a line here to explain the logic of what distribution id is returned based on published and draft scenarios?